spec/models/metric/statistic_spec.rb

```diff
@@ -5,7 +5,7 @@
                          :zone => Zone.first)
     end
 
-    hour = Time.parse(Metric::Helper.nearest_hourly_timestamp(Time.now)).utc
+    hour = Time.now.utc.change(:min => 0)
```

Comment:
This doesn't clear out the seconds or subseconds like nearest_hourly_timestamp does